gradio/components.py

```diff
@@ -1977,7 +1679,7 @@ def style(
 
 
 @document("change", "clear", "play", "pause", "stop", "stream", "style")
-class Audio(Changeable, Clearable, Playable, Streamable, IOComponent):
+class Audio(Changeable, Clearable, Playable, Streamable, IOComponent, FileSerializable):
```

Comment:
sst_or_tts is broken when you cache examples on the `tts_demo` interface

### Changes to demo
```python
import gradio as gr

title = "GPT-J-6B"

tts_examples = [
    "I love learning machine learning",
    "How do you do?",
]

tts_demo = gr.Interface.load(
    "huggingface/facebook/fastspeech2-en-ljspeech",
    title=None,
    examples=tts_examples,
    description="Give me something to say!",
    cache_examples=True
)

stt_demo = gr.Interface.load(
    "huggingface/facebook/wav2vec2-base-960h",
    title=None,
    inputs="mic",
    description="Let me try to guess what you're saying!",
)

demo = gr.TabbedInterface([tts_demo, stt_demo], ["Text-to-speech", "Speech-to-text"])

if __name__ == "__main__":
    demo.launch()
```

### exception
```
❯ python demo/sst_or_tts/run.py
Fetching model from: https://huggingface.co/facebook/fastspeech2-en-ljspeech
Caching examples at: '/Users/freddy/sources/gradio/gradio_cached_examples/51/log.csv'
Traceback (most recent call last):
  File "demo/sst_or_tts/run.py", line 10, in <module>
    tts_demo = gr.Interface.load(
  File "/Users/freddy/sources/gradio/gradio/interface.py", line 109, in load
    return super().load(name=name, src=src, api_key=api_key, alias=alias, **kwargs)
  File "/Users/freddy/sources/gradio/gradio/blocks.py", line 759, in load
    return external.load_blocks_from_repo(name, src, api_key, alias, **kwargs)
  File "/Users/freddy/sources/gradio/gradio/external.py", line 41, in load_blocks_from_repo
    blocks: gradio.Blocks = factory_methods[src](name, api_key, alias, **kwargs)
  File "/Users/freddy/sources/gradio/gradio/external.py", line 306, in get_models_interface
    interface = gradio.Interface(**kwargs)
  File "/Users/freddy/sources/gradio/gradio/interface.py", line 595, in __init__
    self.examples_handler = Examples(
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 51, in create_examples
    utils.synchronize_async(examples_obj.create)
  File "/Users/freddy/sources/gradio/gradio/utils.py", line 362, in synchronize_async
    return fsspec.asyn.sync(fsspec.asyn.get_loop(), func, *args, **kwargs)
  File "/Users/freddy/miniconda3/envs/gradio/lib/python3.8/site-packages/fsspec/asyn.py", line 96, in sync
    raise return_result
  File "/Users/freddy/miniconda3/envs/gradio/lib/python3.8/site-packages/fsspec/asyn.py", line 53, in _runner
    result[0] = await coro
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 194, in create
    await self.cache_interface_examples()
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 231, in cache_interface_examples
    raise e
  File "/Users/freddy/sources/gradio/gradio/examples.py", line 228, in cache_interface_examples
    cache_logger.flag(prediction)
  File "/Users/freddy/sources/gradio/gradio/flagging.py", line 163, in flag
    component.deserialize(
  File "/Users/freddy/sources/gradio/gradio/serializing.py", line 129, in deserialize
    if x.get("is_file", False):
AttributeError: 'str' object has no attribute 'get'
```

Reply:
I think the problem is that the external.py module is still returning base64 representation of the audio?

Reply:
Ah okay I see, I think I missed this workflow: caching examples in Interfaces loaded from external.py

Reply:
Should be fixed now!